float64 qr_long
float64 qr_lat
float64 qr_alt
